server/Dockerfile

```diff
@@ -26,10 +26,6 @@ COPY server ./server
 COPY quests ./quests
 
 ENV NODE_ENV=production
-RUN yarn workspace @linux-odyssey/file-graph build
-RUN yarn workspace @linux-odyssey/utils build
-RUN yarn workspace @linux-odyssey/models build
-RUN yarn workspace @linux-odyssey/constants build
```

Comment:
Are these commands combined and moved to other place?

Reply:
Yes, by using project references, all the dependency packages are built by `tsc -b`